docs/.link-check-errors.txt

```diff
@@ -1,5 +1,4 @@
 Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
-- https://mui.com/joy-ui/integrations/material-ui/
```

Comment:
It's failing [on master](https://app.circleci.com/pipelines/github/mui/material-ui/107013/workflows/1148282e-9ecf-4141-8aad-0ad1d85543d0/jobs/572836), just fixing it here to unblock my CI 😓

Reply:
It's been fixed already. Please merge in the latest master.

Reply:
Updated!